lib/D2L.Extensibility.AuthSdk/D2L.Extensibility.AuthSdk.csproj

```diff
@@ -1,107 +1,15 @@
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <CodeAnalysisRuleSet>AllRules.ruleset</CodeAnalysisRuleSet>
-  </PropertyGroup>
-  <ItemGroup>
-    <Reference Include="System" />
-    <Reference Include="System.Core">
-      <RequiredTargetFramework>3.5</RequiredTargetFramework>
-    </Reference>
-    <Reference Include="System.Web" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="..\GlobalAssemblyInfo.cs" />
-    <Compile Include="D2LAppContextFactory.cs" />
-    <Compile Include="HostSpec.cs" />
-    <Compile Include="Impl\D2LAppContext.cs" />
-    <Compile Include="Impl\D2LUserContext.cs" />
-    <Compile Include="Impl\D2LSigner.cs" />
-    <Compile Include="Impl\DefaultTimestampProvider.cs" />
-    <Compile Include="ID2LAppContext.cs" />
-    <Compile Include="ID2LUserContext.cs" />
-    <Compile Include="Impl\TimestampParser.cs" />
-    <Compile Include="InternalsVisibleToTests.cs" />
-    <Compile Include="ITimestampProvider.cs" />
-    <Compile Include="RequestResult.cs" />
-    <Compile Include="UserContextProperties.cs" />
-    <Compile Include="D2LWebException.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <Folder Include="Properties\" />
-  </ItemGroup>
-  <ItemGroup>
-    <BootstrapperPackage Include="Microsoft.Net.Client.3.5">
-      <Visible>False</Visible>
-      <ProductName>.NET Framework 3.5 SP1 Client Profile</ProductName>
-      <Install>false</Install>
-    </BootstrapperPackage>
-    <BootstrapperPackage Include="Microsoft.Net.Framework.3.5.SP1">
-      <Visible>False</Visible>
-      <ProductName>.NET Framework 3.5 SP1</ProductName>
-      <Install>true</Install>
-    </BootstrapperPackage>
-    <BootstrapperPackage Include="Microsoft.Windows.Installer.3.1">
-      <Visible>False</Visible>
-      <ProductName>Windows Installer 3.1</ProductName>
-      <Install>true</Install>
-    </BootstrapperPackage>
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
-</Project>
\ No newline at end of file
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFramework>netstandard2.0</TargetFramework>
+
+		<Title>D2L.Extensibility.AuthSdk</Title>
+		<Product>D2L.Extensibility.AuthSdk</Product>
+		<Description>D2L.Extensibility.AuthSdk</Description>
+		<Version>0.147.0</Version>

```

Comment:
Probably a different version. `2.0.0`?

Reply:
Lol ya mean to do that, thanks.
You can tell which project I copied this stuff from :)